defaults/main.yml

```diff
@@ -3,16 +3,18 @@
 # Enable or disable PKI support
 pki: True
 
+# Default domain
+pki_domain: '{{ ansible_domain }}'
```

Comment:
@drybjed On mine, `ansible_domain` was set, but it was already blank. Is there a difference between `default('")` and `d("")`?